### ISSUE TITLE
Add external identifier (ror or fundref) separately?

### ISSUE THREAD

**Issue by nicolasfranck** [DMPRoadmap/roadmap#2931]
Please complete the following fields as applicable:

**What version of the DMPRoadmap code are you running? (e.g. v2.2.0)**

3.0.2

**Expected behaviour:**

Add ror or fundref identifier separately to an org

**Actual behaviour:**

It is only indirectly possible to add such identifiers (org_selectors/external_only),
only during creation of an org.

**Steps to reproduce:**

**Comment by briri** [comment]
Yes, this would be a good enhancement. We had initially allowed for this but then removed it once we discovered an issue with the user trying to disambiguate results from the API for institutions with same/similar names. For example, when searching for 'Lincoln University' you would get back 3+ results which each represented a unique institution but no way for the user to determine which one was the correct entry.

It looks like this has been addressed though on their end and they now include a region to help distinguish the records.
See: https://ror.org/search?page=1&query=lincoln%20university
The second issue, which I think is no longer a concern, was having a user try to disambiguate results with similar
